Uplift/Old Files/OldGymDetailViewController.swift

```diff
@@ -0,0 +1,685 @@
+            }
+        } 
+        else {
+            return 0
+        }
+    }
+
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        if tableView == hoursTableView {
+            let cell = tableView.dequeueReusableCell(withIdentifier: GymHoursCell.identifier, for: indexPath) as! GymHoursCell
+            let date = Date()
+            let day = (date.getIntegerDayOfWeekToday() + indexPath.row + 1) % 7
+
+            cell.hoursLabel.text = getStringFromDailyHours(dailyGymHours: gym.gymHours[day])
+
+            switch days[day] {
+            case .sunday:
+                cell.dayLabel.text = "Su"
+            case .monday:
+                cell.dayLabel.text = "M"
+            case .tuesday:
+                cell.dayLabel.text = "T"
+            case .wednesday:
+                cell.dayLabel.text = "W"
+            case .thursday:
+                cell.dayLabel.text = "Th"
+            case .friday:
+                cell.dayLabel.text = "F"
+            case .saturday:
+                cell.dayLabel.text = "Sa"
+            }
+
+            return cell
+        } else {
+            return UITableViewCell()
+        }
+    }
+
+    func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
+        if tableView == hoursTableView {
+            let header = tableView.dequeueReusableHeaderFooterView(withIdentifier: GymHoursHeaderView.identifier) as! GymHoursHeaderView
+
+            header.hoursLabel.text = getStringFromDailyHours(dailyGymHours: gym.gymHoursToday)
+
+            let gesture = UITapGestureRecognizer(target: self, action: #selector(self.dropHours(sender:) ))
+            header.addGestureRecognizer(gesture)
+
+            return header
+        } else {
+            return nil
+        }
+    }
+}
+
+// MARK: TableViewDelegate
+extension OldGymDetailViewController: UITableViewDelegate {
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        if tableView == hoursTableView {
+            return indexPath.row == 5 ? 19 : 27
+        } else {
+            return 112
+        }
+    }
+
+    func tableView(_ tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
+        if tableView == hoursTableView {
+            if (hoursData.isDropped) {
+                return 27
+            } else {
+                return 19
+            }
+        } else {
+            return 0
+        }
+    }
+}
+
+// MARK: - ScrollViewDelegate
+extension OldGymDetailViewController: UIScrollViewDelegate {
+    func scrollViewDidScroll(_ scrollView: UIScrollView) {
+        statusBarUpdater?.refreshStatusBarStyle()
+
+        switch UIApplication.shared.statusBarStyle {
+        case .lightContent:
+            backButton.setImage(UIImage(named: "back-arrow"), for: .normal)
+        case .default:
+            backButton.setImage(UIImage(named: "darkBackArrow"), for: .normal)
```

Comment:
yeah we should move `back-arrow` and `darkBackArrow` into a global Constants file